Directory.Build.props

```diff
@@ -151,9 +151,7 @@
       Include="$(RepoRoot)src\Analyzers\Internal.AspNetCore.Analyzers\src\Internal.AspNetCore.Analyzers.csproj"
       ReferenceOutputAssembly="false"
       OutputItemType="Analyzer"
-      PrivateAssets="All"
-      Version="$(InternalAspNetCoreAnalyzersPackageVersion)"
-      IsImplicitlyDefined="true" />
+      PrivateAssets="All" />
```

Comment:
We get the versions from `dependencies.props` now, yeah? https://github.com/dotnet/aspnetcore/blob/5782872f7f79628d48dc88f0f60ff249fe589fe6/eng/Dependencies.props#L189-L196

(For my understanding, not asking for a change)

Reply:
No, `%(Version)` metadata doesn't apply to project references and Dependencies.props only contains information about package references.

But also yes, versions for all `@(Reference)` items transformed into `@(PackageReference)` items come from Dependencies.props and have forever. The code in your link is about calculating the `@(LatestPackageReference)` metadata automatically and added support for `%(RTMVersion)` metadata.

Reply:
Ah, right, forgot this was a ProjectReference